storage/test/storage/ArrayStorageTest.java

```diff
@@ -0,0 +1,93 @@
+package storage;
+
+import model.Resume;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import storage.exception.ResumeAlreadyStoredException;
+import storage.exception.ResumeNotFoundException;
+
+abstract class ArrayStorageTest {
```

Comment:
Run the test with coverage and cover missing parts. For example, there is no check for storage overflow. No testing of the `update` method etc.

Reply:
Done